Setup/UI/RequiredSoftwareDlg.wxs

```diff
@@ -21,11 +21,7 @@
         <Control Id="KDiffTitle" Type="Text" Height="12" Width="200" X="15" Y="111" Text="{\WixUI_Font_Title}KDiff3 32bit 0.9.97" />
         <Control Id="KDiffDescription" Type="Text" Height="44" Width="238" X="25" Y="127" Text="KDiff is a diff/merge tool that Git Extensions calls upon when it encounters a merge conflict. If you already have a diff/merge tool that works with Git then you don't need to install this." />
         <Control Id="KDiffCheckBox" Type="CheckBox" Height="20" Width="78" X="274" Y="129" Text="Install KDiff" Property="INSTALLKDIFF3" CheckBoxValue="1" />
-
-        <Control Id="GitCredentialWinStoreTitle" Type="Text" Height="12" Width="200" X="15" Y="176" Text="{\WixUI_Font_Title}Windows Credential Store for Git 1.4" />
```

Comment:
Just plain text editor

It should look like
![install2 1](https://cloud.githubusercontent.com/assets/224272/14736108/92adabba-087d-11e6-9fd1-c8c7552855bc.png)

After your changes it would be like
![test](https://cloud.githubusercontent.com/assets/224272/14736180/ffc53e84-087d-11e6-8859-70f794353938.png)